<launch>

    <param name="use_sim_time" value="true" />

    <!-- event rendering -->
    <node name="dvs_renderer" pkg="dvs_renderer" type="dvs_renderer">
        <param name="display_method" value="grayscale"/>
        <!-- <param name="display_method" value="red-blue"/> -->
        <remap from="events" to="/dvs/events" />
        <remap from="camera_info" to="/dvs/camera_info" />
    </node>

    <arg name="config_file" default="$(find eloam)/config/config_eloam.yaml" />
    <arg name="result_save" default="true" />
    <arg name="output_path" default="/home/jjiao/catkin_ws/src/localization/rpg_trajectory_evaluation/results/xxx/" />
    <arg name="data_source" default="bag" />
    
    <node pkg="eloam" type="eloam_node" name="eloam_node"
        args="-config_file=$(arg config_file)
                -result_save=$(arg result_save) 
                -output_path=$(arg output_path)
                -data_source=$(arg data_source)" output="screen">

        <remap from="/events" to="/dvs/events" />
        <remap from="/image_raw" to="/dvs/image_raw" />
        <remap from="/camera_info" to="/dvs/camera_info" />
    </node>

 </launch>
